### PR TITLE
Test for aktualizr-repo scripts

### PR DESCRIPTION
**Pull request by OYTIS**

### CODE REVIEW

File: src/aktualizr_repo/run/create_repo.sh [hunk 1]
```diff
@@ -19,11 +19,17 @@ distinguished_name = req_distinguished_name
 basicConstraints = critical,CA:true
 keyUsage = keyCertSign
 EOF
+        # CA certificate for client devices
         openssl genrsa -out ${certs_dir}/client/ca.private.pem 4096
         openssl req -key ${certs_dir}/client/ca.private.pem -new -x509 -days 7300 -out ${certs_dir}/client/cacert.pem -subj "/C=DE/ST=Berlin/O=Reis und Kichererbsen e.V/commonName=aktualizr-repo-client-ca" -batch -config ${certs_dir}/ca.cnf -extensions cacert
+	# certificate for stand-alone device
+	openssl req -out ${certs_dir}/client/standalone_device_cert.csr -subj "/C=DE/ST=Berlin/O=Reis und Kichererbsen e.V/commonName=sota_device" -batch -new -newkey rsa:2048 -nodes -keyout ${certs_dir}/client/standalone_device_key.pem
+	openssl x509 -req -in ${certs_dir}/client/standalone_device_cert.csr -CA ${certs_dir}/client/cacert.pem -CAkey ${certs_dir}/client/ca.private.pem -CAcreateserial -out ${certs_dir}/client/standalone_device_cert.pem
```
Comment (patrickvacek) [inline src/aktualizr_repo/run/create_repo.sh]: Some weird tab/spacing mismatches, just FYI.

File: scripts/test_aktualizr_repo.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
+
+set -exuo pipefail
+
+export PATH=${PATH}/persistent
+
+TMPDIR=$(mktemp)
+create_repo.sh $TMPDIR 127.0.0.1
+serve_repo.py 9000 $TMPDIR
```
[the remark this answers is not in the document]
Comment (OYTIS) [reply scripts/test_aktualizr_repo.sh]: Well, right now 9000 is hardcoded in `create_repo.sh`. Might be changed, I just wanted to pass as few parameters as possible, but this is too few maybe.

**Comment by OYTIS** [comment]
This feature in `aktualizr-repo` that makes it wait till device is provisoned was never implemented, right?

**Comment by patrickvacek** [comment]
@OYTIS, nope. PRO-4820 in the old Jira. It's still on my backlog.

**Comment by OYTIS** [comment]
Sorry, I completely forgot about 'new' running modes, that's what I really need here.

**Pull request merged**